examples/src/main/java/io/kubernetes/client/examples/PatchExample.java

```diff
@@ -102,17 +101,17 @@ public static void main(String[] args) throws IOException {
       // apply-yaml a deployment, server side apply is alpha in kubernetes v1.14,
       // You need to actively enable the Server Side Apply alpha feature
       // https://kubernetes.io/docs/reference/using-api/api-concepts/#server-side-apply
-      ExtensionsV1beta1Deployment deploy4 =
+      V1Deployment deploy4 =
           PatchUtils.patch(
```

Comment:
@yue9944882 I run your version of example and for yaml apply I received an error:

{"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"failed to create typed patch object: .spec.template.spec.containers[name=\"hello-node\"].ports: element 0: associative list with keys has an element that omits key field \"protocol\"","code":500}

Reply:
can you retry on my latest update? i just re'produced the error on a v1.18 cluster and fixed it.

Reply:
thank you, now it works